content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -193,8 +193,13 @@ The following diagram represents an example of this kind of setup:
 
 In this case, you can replace the Git metadata reported in the deployment with the metadata of the application repository instead of the configuration repository. This allows you to connect the deployments performed by Argo CD and the related CI pipeline runs on the application repository.
 
-To associate the application repository Git information with the Argo CD deployments, run the `datadog-ci deployment correlate` command between committing and pushing the changes to the configuration repository. This requires the `datadog-ci` CLI version to be `2.41.0` or later. See the [command syntax][14] for additional details:
+To associate the application repository Git information with the Argo CD deployments, run the `datadog-ci deployment correlate` in your CI. The required command arguments vary by the setup:
+* **Direct push**: If the CI pipeline directly pushes the changes to the configuration repository and triggers an Argo CD deployment.
+* **Manually**: If the CI pipeline does not directly push the changes to the configuration repository. For example: the CI creates a feature branch, commits to it and lastly merges it.
```

Comment:
For `1`, I would not say it only works with the default branch only. For example, you can push your changes to a feature branch and If your ArgoCD application monitors feature branches, it will trigger a deployment and the correlation will work.